src/main/java/org/spongepowered/api/item/inventory/custom/CustomInventory.java

```diff
@@ -54,7 +55,7 @@ static Builder builder() {
          * @param name The title
          * @return fluent pattern
          */
-        Builder name(Translation name);
+        Builder name(String name);
```

Comment:
Find a way to change that method to return the `IChatComponent` from the `Text` object (`SpongeTexts.toComponent`). I assume the protocol allows for sending a text object.